config/locales/en.yml

```diff
@@ -1,33 +1,10 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  errors:
+    server: 'An error ocurred'
+    record_not_found: 'No such record'
+    missing_param: 'The following param is missing or the value is empty: %{param}'
+    invalid_content_type: 'Invalid content type header'
+    authentication:
+      authentication_needed: 'Authentication is required to perform this action'
+      invalid_credentials: 'Credentials not valid'
```

Comment:
Use full errors, since they are going to be shown to the final user. 
`Invalid credentials`, or `The credentials are not valid`, same for the others